hera_mc/tests/test_sysutils.py

```diff
@@ -236,40 +199,41 @@ def test_correlator_info(sys_handle):
 
 def test_get_pam_from_hookup(sys_handle, mcsession):
     hookup = cm_hookup.Hookup(mcsession)
-    hud = hookup.get_hookup(['HH23'], at_date='2017-07-03', exact_match=True,
-                            hookup_type='parts_paper')
+    hud = hookup.get_hookup(['HH701'], at_date='2019-07-03', exact_match=True,
+                            hookup_type='parts_hera')
     pams = hud[list(hud.keys())[0]].get_part_in_hookup_from_type(
         'post-amp', include_ports=True, include_revs=True)
     assert len(pams) == 2
     # the actual pam number (the thing written on the case)
-    assert pams['e'] == 'ea>PAM75123:B<eb'
+    assert pams['E<ground'] == 'e>PAM701:A<e'
 
 
 def test_get_pam_info(sys_handle, mcsession):
     sys_h = cm_sysutils.Handling(mcsession)
     pams = sys_h.get_part_at_station_from_type(
-        'HH23', '2017-07-03', 'post-amp', include_ports=False,
-        hookup_type='parts_paper')
+        'HH701', '2019-07-03', 'post-amp', include_ports=False,
+        hookup_type='parts_hera')
     assert len(pams) == 1
     # the actual pam number (the thing written on the case)
-    assert pams['HH23:A']['e'] == 'PAM75123'
+    assert pams['HH701:A']['E<ground'] == 'PAM701'
 
 
 def test_apriori_antenna(sys_handle, mcsession, capsys):
-    cm_partconnect.update_apriori_antenna('HH2', 'needs_checking',
-                                          '1214482618', session=mcsession)
+    cm_partconnect.update_apriori_antenna('HH701', 'needs_checking',
+                                          '1214382618', session=mcsession)
     d = sys_handle.get_apriori_antenna_status_for_rtp('needs_checking')
-    assert d == 'HH2'
+    assert d == 'HH701'
     g = sys_handle.get_apriori_antenna_status_set()
-    assert g['needs_checking'][0] == 'HH2'
+    assert g['needs_checking'][0] == 'HH701'
     pytest.raises(ValueError, cm_partconnect.update_apriori_antenna,
                   'HH3', 'not_one', '1214482618', session=mcsession)
-    g = sys_handle.get_apriori_status_for_antenna('HH2')
+    g = sys_handle.get_apriori_status_for_antenna('HH701')
     assert g == 'needs_checking'
     cm_partconnect.update_apriori_antenna(
-        'HH2', 'needs_checking', '1214482818', '1214483000', session=mcsession)
-    pytest.raises(ValueError, cm_partconnect.update_apriori_antenna,
-                  'HH2', 'needs_checking', '1214482838', session=mcsession)
+        'HH700', 'needs_checking', '1214482818', '1214483000',
+        session=mcsession)
+    pytest.raises(ValueError, cm_partconnect.update_apriori_antenna, 'HH700',
+                  'needs_checking', '1214482838', session=mcsession)
     print(cm_partconnect.AprioriAntenna())
```

Comment:
I think this print statement should come out.

Reply:
It is there to to be captured by the next line to check the code.

Reply:
Interesting approach. Ok.